### DISCUSSION TITLE
New addon : "Costume editor improvement"

### DISCUSSION THREAD

**Discussion by Medericoder** [ScratchAddons/ScratchAddons#3402] [Ideas]
Hi!
Have a look on this :
![image](https://user-images.githubusercontent.com/80577386/131320899-d1ee108a-e0e2-4466-ba27-0ed6d9941608.png)

I think you can add an addon : "Costume editor improvement" to add buttons :

- New shapes (triangle, star, heart...), it may be a drop-down list.
- A color memory (like Scratch 2) that memorizes colors between costumes/sprites even if you click on a colored shape.
- A "shapes backpack" in which you can place custom shapes to use it easier.

**Comment by VeryUsual** [comment]
That would be great

**Comment by Marc92020** [comment]
That would be cool, but new shapes buttons might be hard/impossible
Also shapes backpack could be possible but it might be really hard

**Comment by Endermaster3690** [comment]
Also aligning text to the right and middle. Usually text is aligned to the left.

**Reply by Medericoder** [reply]
Why not!
